base/src/main/java/butter/droid/base/providers/media/VodoProvider.java

```diff
@@ -256,6 +265,8 @@ private int isInResults(ArrayList<Media> results, String id) {
                 movies = downloads;
             }
 
+            ArrayList<Media> items = new ArrayList<>();
```

Comment:
`MediaProvider` at the moment works with `ArrayList` and not `List`. This will be changed as part provider rewrite.